sonar-plugin-api/src/main/java/org/sonar/api/Plugin.java

```diff
@@ -85,18 +85,29 @@
 public interface Plugin {
 
   class Context {
-    private final Version version;
+    private final Version runtimeApiVersion;
     private final List extensions = new ArrayList();
+    private final boolean sonarlintRuntime;
 
-    public Context(Version version) {
-      this.version = version;
+    public Context(Version runtimeApiVersion, boolean sonarlintRuntime) {
+      this.runtimeApiVersion = runtimeApiVersion;
+      this.sonarlintRuntime = sonarlintRuntime;
     }
 
     /**
-     * Runtime version of SonarQube
+     * @deprecated since 6.0
      */
+    @Deprecated
     public Version getSonarQubeVersion() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)